databricks/sdk/core.py

```diff
@@ -615,6 +615,10 @@ def wrap_debug_info(self, message: str) -> str:
         debug_string = self.debug_string()
         if debug_string:
             message = f'{message.rstrip(".")}. {debug_string}'
+        elif self._credentials_provider.auth_type() == "default":
+            # Empty debug string when auth type is default means we are likely missing some files or environment variables. Add a debug string to make it easier to understand.
+            default_auth_prompt = "Please check https://github.com/databricks/databricks-sdk-py/tree/main#default-authentication-flow to configure credentials for your preferred authentication method."
+            message = f'{message.rstrip(".")}. {default_auth_prompt}'
```

Comment:
Added this prompt for cases that debug string is empty while using default auth.